packages/flutter/lib/src/material/reorderable_list.dart

```diff
@@ -65,6 +65,43 @@ import 'theme.dart';
 /// ```
 ///
 ///{@end-tool}
+///
+/// {@tool snippet}
+/// This example mirrors the previous one, creating the same list using the
+/// [ReorderableListView.builder] constructor. Using the [IndexedWidgetBuilder], The
+/// list items are built lazily on demand.
```

Comment:
If this is moved to the `.builder` constructor, this would probably be better phrased as:

```
/// This example creates a list using the [ReorderableListView.builder]
/// constructor. By providing a [IndexedWidgetBuilder], the list items
/// are built lazily on demand, which will improve scrolling performance
/// for large lists.
```

Reply:
moved it to .builder constructor

Reply:
It would probably still be a good idea to change the 'This example mirrors the previous one,' part as they may be just reading this example as it is no longer part of the other comment and wouldn't have the context for it.

Reply:
done